### DISCUSSION TITLE
Shutter pwm slider

### DISCUSSION THREAD

**Discussion by wollo-fpv** [arendst/Tasmota#15581] [Q&A] [answered]
Hello,

I have opend a question....
https://github.com/arendst/Tasmota/discussions/15516
Like I have written before, its working well!
I can set the range of rpm's of the motor using the device interface page, with the slider.
https://raw.githubusercontent.com/TrDA-hab/Projects/master/Servo%2BESP8266/511.jpg
I have add the device to home assistant.
Here I see only a two state button, open and close.
https://drive.google.com/file/d/1zdazOHgGIYG48eLE7W0VTPTsN9UujcVD/view?usp=sharing
I hoped that I could control the pwm from 0% to 100% (Like a dimmer)
And to put some automations in home assistant.
example, if temperature is 25°C that the fan have to spin with 10%, if temperature is 30°C to spin with 40%, etc
What need to be changed, that the pwm slider on the device page will be enabled and working in Home Assistant.

Wolf

**Comment by TrDA-hab** [comment, marked as answer]
You need:
1. Sonoff D1 Dimmer
2. Dimmable AC motor
https://www.youtube.com/watch?v=S4nMcrPoYII&feature=youtu.be

**Comment by wollo-fpv** [comment]
Sorry, 
No.......
It's for DC
I need to use the pwm control for rc servo's.
https://github.com/arendst/Tasmota/discussions/10443

Is working well with my dc motor and a rc speed controller.

I don't get the pwm (rpm) slider into Home Assistant .

Wolf
